docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -490,6 +490,14 @@ spec:
 > default interface type to `masquerade`, and disabling the `bridge`
 > type for pod network, as shown in the example above.
 
+> **Note:** Migrating a VM with secondary interfaces that uses bridge 
+> binding may cause long periods of traffic disruption. This occurs when
+> the interface is defined with an explicit MAC address (manually or 
+> automatically through KubeMacPool) on nodes that have IPv6 enabled.
+> In order to mitigate traffic disruption during live migration consider 
```

Comment:
"If bridge CNI is used, it is possible to avoid this issue by setting..."

Reply:
Note that this note is under `bridge` section

Reply:
under bridge binding section. Bridge ibnding can be used with various CNIs (bridge-cni, ovs-cni, ovn-kube, etc...)

Reply:
Done